src/python_testing/TC_SU_2_2.py

```diff
@@ -0,0 +1,844 @@
+                provider_setupPinCode=provider_setupPinCode,
+                secured_device_port=provider_port,
+                queue="updateAvailable",
+            )
+        else:
+            provider_proc = await self.launch_ota_provider(
+                ota_file=provider_ota_file,
+                provider_discriminator=provider_discriminator,
+                provider_setupPinCode=provider_setupPinCode,
+                secured_device_port=provider_port,
+                queue=provider_queue,       # STEP_2
+                timeout=provider_timeout    # STEP_2
+            )
+
+        # Prerequisite #2.1 - Open commissioning window on DUT
+        logger.info(f'{step_number}: Prerequisite #2.1 - Commissioning window opened on Requestor (DUT)')
+
+        params = await self.open_commissioning_window(controller, requestor_node_id)
+        setup_pin_code = params.commissioningParameters.setupPinCode
+        long_discriminator = params.randomDiscriminator
+        logging.info(f'{step_number}: Prerequisite #2.1 - Commissioning window opened: {vars(params)}')
+
+        # Prerequisite #2.2 - Commission Provider
+        logging.info(f'{step_number}: Prerequisite #2.2 - Commissioning DUT with Provider')
+
+        resp = await controller.CommissionOnNetwork(
+            nodeId=provider_node_id,
+            setupPinCode=provider_setupPinCode,
+            filterType=ChipDeviceCtrl.DiscoveryFilterType.LONG_DISCRIMINATOR,
+            filter=provider_discriminator
+        )
+        logger.info(f'{step_number}: Prerequisite #2.2 - Provider Commissioning response: {resp}')
+
+        # Prerequisite #3.0 - Setting ACLs
+        logger.info(f'{step_number}: Prerequisite #3.0 - Setting ACLs under FabricIndex {fabric_id}')
+        logger.info(f'{step_number}: Prerequisite #3.0 - Requestor (DUT), NodeID: {requestor_node_id}')
+        logger.info(f'{step_number}: Prerequisite #3.0 - Provider, NodeID: {provider_node_id}')
+
+        # Set ACLs for OTA update:
+        # - On Requestor to allow specified Provider to interact with OTA Requestor cluster
+        # - On Provider to allow Requestor to interact with OTA Provider cluster
+        await self.set_ota_acls_for_provider(
+            controller,
+            requestor_node=requestor_node_id,
+            provider_node=provider_node_id,
+            fabric_index=fabric_id
+        )
+
+        # Prerequisite #4.0 - Add OTA Provider to the Requestor
+        logger.info(f'{step_number}: Prerequisite #4.0 - Add Provider to Requestor(DUT) DefaultOTAProviders')
+
+        await self.add_single_ota_provider(controller, requestor_node_id, provider_node_id)
+
+        return requestor_proc, provider_proc
+
+    def desc_TC_SU_2_2(self) -> str:
+        return "[TC-SU-2.2] Handling Different QueryImageResponse Scenarios on Requestor"
+
+    def pics_TC_SU_2_2(self):
+        """Return the PICS definitions associated with this test."""
+        pics = [
+            "MCORE.S",      # Pics
```

Comment:
```suggestion
            "MCORE.OTA.Requestor",      # Pics
```
But actually, this should properly be OTAR.S since we should be running this test on every device with an ota requestor cluster. It's just wrong in the test plan right now. Or...not wrong, just old.

Reply:
Updated: Updated PICS to `MCORE.OTA.Requestor` as suggested. Aware that long term this should be `OTAR.S`